### PR TITLE
Isolation Glossary

### PR DESCRIPTION
**Pull request by browkin**
- Added Isolation to Glossary
- Linked each other

glossar1
glossar2

RKI Link for 'Kriterien' in Isolation : https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Entlassmanagement.html?nn=13490888

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@browkin 
Will you add an English translation?

If there is no English text available, then perhaps add the terms
- Isolation 
- Quarantine

to the English glossary and link to the German glossary with a note that the definitions are only available in German?

**Comment by dsarkar** [comment]
@MikeMcC399 FYI EN texts been added.

File: src/data/faq.json [hunk 1]
```diff
@@ -2545,6 +2545,11 @@
                     "term": "iOS",
                     "anchor": "ios",
                     "description": "iOS is a mobile operating system developed by Apple for the iPhone, iPod touch and formerly also iPad. Along with <a href='#glossary_android'>Android</a>, it is one of the two operating systems that the Corona-Warn-App supports."
+                },{
+                    "term": "Isolation",
+                    "anchor": "isolation",
+                    "description": "Isolation is a government-ordered measure for persons who have been confirmed to have SARS-CoV-2 infection by a PCR-Test.Isolation can be gone through at home or, if the disease is severe, in a hospital. Release from isolation is based on specified <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Entlassmanagement.html?nn=13490888'> criteria (German only)</a> . At the time of release, the person is considered to no longer be infectious. See also <a href='#glossary_quarantine'>Quarantine</a>. "
```
Comment (MikeMcC399) [inline src/data/faq.json:2551]: Whitespace corrections:

```suggestion
                    "description": "Isolation is a government-ordered measure for persons who have been confirmed to have SARS-CoV-2 infection by a PCR-Test. Isolation can be gone through at home or, if the disease is severe, in a hospital. Release from isolation is based on specified <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Entlassmanagement.html?nn=13490888'> criteria (German only)</a>. At the time of release, the person is considered to no longer be infectious. See also <a href='#glossary_quarantine'>Quarantine</a>. "
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Inline suggestions to correct whitespace.

**Comment by MikeMcC399** [comment]
@dsarkar 
> FYI EN texts been added.

Thanks for the heads-up! The text looks fine, just minor formatting to be fixed.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the review and corrections.

**Pull request merged**